examples/modus_toolbox/golioth_basics/golioth_app/Makefile

```diff
@@ -225,6 +228,7 @@ DEFINES+=APP_VERSION_MAJOR=$(APP_VERSION_MAJOR) \
          USER_APP_SIZE=$(SLOT_SIZE)
 DEFINES+=CONFIG_GOLIOTH_USER_CONFIG_INCLUDE='"golioth_user_config.h"'
 DEFINES+=HEATSHRINK_DYNAMIC_ALLOC=0
+DEFINES+=MINIZ_NO_MALLOC=1
```

Comment:
Because `miniz_export.h` is a copy-paste of a generated file from a build of standalone `miniz`, and it didn't include configuration values like that.

But now that you mention it, `miniz_export.h` is a more convenient place to define configuration values (one place instead of multiple for each build system).